README.md

```diff
@@ -224,7 +244,7 @@ Below you can see a schematic overview of the different concepts present in the
 
 
 ### Next Steps
-Head to the [documentation](https://docs.lightly.ai) and see the things you can achieve with Lightly!
+Head to the [documentation](https://docs.lightly.ai) and see the things you can achieve with lightly!
```

Comment:
Lightly big?

Reply:
Would also keep this lower as it's OSS related